lib/commands/hasFocus.js

```diff
@@ -27,19 +27,21 @@
  */
 
 let hasFocus = function (selector) {
-    let result = this.execute(function (selector) {
-        var focused = document.activeElement
+    return this.elements(selector).then((res) => {
```

Comment:
again: no need to call elements if you only pass the selector into the execute function anyway